include/SSVOpenHexagon/Data/StyleData.hpp

```diff
@@ -96,7 +99,8 @@ class StyleData
               ssvuj::getExtr<float>(mRoot, "3D_perspective_multiplier", 1.f)},
           _3dOverrideColor{ssvuj::getExtr<sf::Color>(
               mRoot, "3D_override_color", sf::Color::Transparent)},
-          mainColorData{ssvuj::getObj(mRoot, "main")}
+          mainColorData{ssvuj::getObj(mRoot, "main")}, //
+          capColor{parseCapColor(ssvuj::getObj(mRoot, "cap_color"))}
```

Comment:
@Morxemplum: this is the new JSON key for the cap color. Needs testing. It can be any of:

```
"cap_color": "main"
"cap_color": "main_darkened"
"cap_color": { "index": 0 }
```

Reply:
No RGB customization? Alright. It works

Reply:
Easy to add. Might be a good task for you to pick up